coffee-frontend/src/utils/loginUtils.js

```diff
@@ -1,26 +1,18 @@
 import { API_BASE_URL, ACCESS_TOKEN } from './constants';
+import axios from 'axios'
+import React, { useState, useEffect, useContext, createContext } from "react";
 
-const request_text = (options) => {
-    const headers = new Headers({
-        'Content-Type': 'application/json',
-    })
-    
-    if(localStorage.getItem(ACCESS_TOKEN)) {
-        headers.append('Authorization', 'Bearer ' + localStorage.getItem(ACCESS_TOKEN))
-    }
 
-    const defaults = {headers: headers};
-    options = Object.assign({}, defaults, options);
-    return fetch(options.url, options)
-    .then(response => {
-        return response.text().then(text => {
-            if(!response.ok) {
-                return Promise.reject(text);
-            }
-            return text;
-        })
-});
-};
+
+export const login = (loginRequest) =>  {
+
+    return axios.post(API_BASE_URL + "/auth/login", loginRequest)
+}
+
+export const signup = (signupRequest) =>  {
+    return axios.post(API_BASE_URL + "/auth/signup", signupRequest)
+}
+
 
 
 const request = (options) => {
```

Comment:
This is redundant now

Reply:
Uhm, why?

Reply:
Because it is replaced with axios, and not used